.github/workflows/docker.yml

```diff
@@ -21,17 +21,18 @@ jobs:
     name: Build Images
     runs-on: ubuntu-latest
     steps:
+    - name: Check out code into the Go module directory
```

Comment:
why the order change here? checkout before go setup?

Reply:
Same here, setup-go wants to cache go modules.  So we need to checkout before setting up, otherwise the go.sum is not available.